### ISSUE TITLE
[Reporting] Backend APIs for weekly resolution metrics

### ISSUE THREAD

**Issue by reichert621** [papercups-io/papercups#532]
**Problem**
Before we can display these metrics in the UI, we'll need to write the backend APIs to query and calculate this data.

**Solution**
We've already done this for "reply time" metrics, so the code will be very similar :) 

Basically, we want to write these methods, but have them compare `conversation.closed_at` (rather than `conversation.first_replied_at`) with `conversation.inserted_at`: https://github.com/papercups-io/papercups/blob/master/lib/chat_api/reporting.ex#L80-L107

Let's call this method something like `seconds_to_close_conversation_metrics_by_week(account_id, filters \\ %{})`

(Where `filters` is a map that can optionally include a `from_date` and `to_date` range)

**Testing**
Please include test coverage for these new methods in your PR!

**Questions, or need help getting started?**
Feel free to ask below, or [ping us on Slack](https://join.slack.com/t/papercups-io/shared_invite/zt-h0c3fxmd-hZi1Zp8~D61S6GD16aMqmg) :)

(You can also check out our [CONTRIBUTING.md](https://github.com/papercups-io/papercups/blob/master/CONTRIBUTING.md))
